packages/react-components/react-card/src/components/Card/__snapshots__/Card.test.tsx.snap

```diff
@@ -4,7 +4,7 @@ exports[`Card renders a default state 1`] = `
 <div>
   <div
     class="fui-Card"
-    data-tabster="{\\"groupper\\":{\\"tabbability\\":2}}"
+    data-tabster="{\\"groupper\\":{}}"
```

Comment:
what is this test testing ?

Reply:
This is just testing DOM structure, the reason it changed is that we don't apply a `focusMode` for Card as default anymore and Tabster modifies the DOM structure.